Summary of recent papers on GWAS and Exome Sequencing for common diseases and Complex Traits:

Hirschhorn and Daly's paper was written when GWAS became feasible due to lowering cost and acceptance by funding agencies. A dense set of SNPs across the genome is genotyped to survey the most common genetic variation for a role in disease or to identify the heritable Quantitative Traits taht are risk factors for disease. A well-powered study was yet to be published at that time. The paper considered rationale for GWAS and issues of their power, efficiency, comprehensiveness, interpretation, and analysis. Two feasible study categories are: candidate-gene studies, which use either association or resequencing approaches, and genome wide studies, which include both linkage mapping and GWAS. 

**Why GWAS**: Linkage studies have been highly successful for finding genes for monogenic 'Mendeilian' diseases. Variants that cause monogenic disorders are rare (due to negative selection), so each segregating disease allele will be found in the same 10-20 cM chromosomal background within each family. These are highly penetrant variants; thus markers within 10-20 cM of the disease-causing alleles will co-segregate with diseaase status.  Genome-wide linkage analysis has also been carried out for common diseases, where the high penetrance of Mendelian diseases may not apply (exceptions: IBD, schizophrenia and T1D - even for these exceptions, only a small portion of the heritability is explained). There are many reasons for this lack of success (the paper lists them). However, studies will still need to progress from a broad region of linkage (~10 Mbp) to a causal gene or genes within that region. Linkage analysis is less powerful for identifying commong genetic variants with modest effect on disease. "Most common diseases and clinically important quantitative traits have complex architectures, for which the phenotype is determined by the sum total of, and/or interaction between, multiple genetic and environmental factors." **Any individual genetic variant will likely have a small effect on disease risk**. Some of the well-established disease alleles could never be detected by linkage analysis. Example: *PPARG* affecting T2D will need 1 M sib pairs to uncover. None of the genome-wide linkage studies identified this gene. Thus, a strategic complement to linkage analysis is desirable. 

*Candidate-gene resequencing studies*: Such case-control studies will be laborious and expensive. *MC4R* is one success story here - which explain a small fraction of severe early-onset obestity. It is a small single-exon gene. *Association studies*: Candidate-gene association studies (based on the biological hypotheses or location  of the candidate within a LD-region)  have identified genes that impact common disease. When fundamental physiologic defects of a disease are unknown, this candidate-gene approach will be indequate. *GWAS approach*: This is an association study that scans most of the genome for causal genetic variants. No need to guess the id of the causal genes. It is an unbiased, yet fairly comprehensive approach. GWAS studies became possible (~2005) because of these advances: (1) the dbSNP database contained ~9M SNPS, including most of ~11M SNPs with minor allele frequencies of 1% or greater. (2) Genotyping technology has become cheaper. In 2015, NIH estimated it at $0.01 per genotype to build 600 M genotypes. (3) Availability of LD patterns on a genome-wide basis from the HapMap project. 

**Markers for genome-wide association studies**: *LD-based markers*: Most of the genome falls into segments of strong LD, within which variants are strongly correlated with each other, and most chromosomes carry one of only a few common Haplotypes. ENCODE (Encyclopedia of DNA Elements) project resequenced chromosomes. The study confirmed the patterns of long segments of strong LD, seen in earlier studies. Because of strong correlation among neighboring SNPs, a few tag SNPs can be chosen based on LD patterns, individually (or in multimarker combinations - haplotypes), that capture most of the common variation within the region. A few hundred SNPs should be adequate. A larger number of tag SNPs may be required in African populations, as they generally contain more variation and less LD. *The missense approach*: Typically one or two missense SNPs are there per gene. Genotyping of only 30K-60K will be required. However, the rationale is open to question. These mutations would be under negative selection, and probably more relevant to highly penetrant rare diseases. The alleles that underlie complex traits more likely have modest effect on disease risk and are more likely to include non-coding regulatory variants, with modest impact on expression levels. *A convenience-based approach*: Select variants based on logistical considerations, such as ease and cost of genotyping. It would not be 'genome-wide'. A low cost way would be to target 400-1000 microsatellites that are typed in linkage studies. Say, we survey an LD block of 50 kbp (*typical LD is only 20 kb*), this would cover 20-50Mbp - ~ 1 to 3% of the genome. So, not a GWAS. Current evidence, as of 2005: HapMap project indicates that 100K markers (1 every 30Kb of the genome) would be sufficient; but this seems to cover less than 50% variants. *Can rare alleles be detected by association methods*: Statistical power to detect causal alleles increasaes with increase in the frequency and penetrance of these alleles; So, a single parameter, such as the phenotype amount and variation in it might be useful. If so, both rare, highly penetrant alleles and common, low-penetrance alleles will be on an equal footing. But rare alleles, even with strong effects, may not be well represented in SNP databases and because tag SNP approaches tag common SNPs (MAF > 5%). *An analogy: High S/N ratio necessarily has a Signal that is much above the noise threshold. A low level signal can be extracted, but requires integrating and filtering to discriminate against the noise*. Evolutionary negative pressure, to optimize the cost of survival, would preclude this possibility in future generations. So, only low cost variants that are more commmon and less penetrant would survive multiple generations. The corollary is that rare alleles with MAF <5% are likely to be recent mutations (because old alleles tend to either disappear or become common), so there would have been less time for recombination and mutation to disrupt the haplotype on which they arose. *For such rare variants with modest effect,  the expression level will be low (and undetectable in a population study),  and ability to detect them is also low (MAF < 5%)*. 

**Strategies to increase efficiency**: These include multi-stage design (as with clinical studies - an initiatl study with high sensitivity with a follow-up study of high specificity) to minimize sample sizes, the use of special populations, and the pooling of samples. *Selection of an appropriate sample size*: A *p*-value of 5 x 10<sup>-8</sup> is typically used - this is equivalent to a *p*-value of 0.05 after a Bonferroni correction for 1 million independent tests. This is a o conservative threshold. to understand better: consider an allele frequency of 15% and an odds ratio of 1.25.  6,000 cases and 6,000 controls will be needed to meet this alpha value and a statistical power of 80% (for a beta of 0.20). Let us say, the *p* value is more liberal at 5 x 10<sup>-2<.sup> (that is, the value of 0.05 used without the Bonferroni correction), one would still need 1,200 cases and 1,200 controls - or 1.2 billion genotypes - would be needed (for 500K independent SNPs per genome). Note: There are *10 million SNPs* in the human genome, one every 300 bases on average. 



Definitions/Elaborations:

* [CM](https://isogg.org/wiki/CentiMorgan): CentiMorgan estimates a chromosomal regiion's to propensity to recombine, as determined from parent-siblings studies.  One centiMorgan = ~ 1 Mbp. 1 CM = 1% chance that a marker at one genetic locus on a chromosome will be separated from a marker at a second locus due to crossing over in a single generation. An analogy: if the terrain is hilly, 1 mile (similar to 1 Mbp)  might feel like 2 miles. Thus cM captures the probability of similarity of the 'terrain'. If two cousins share 40 cM on chromosome 1 (approximately 40 Mbp in length), they can be predicted to share a certain degree of relationship statistically (there is an average 30% chance of crossover for 40 cM). 
* Genetic Marker: It can be a SNP or as long as a minisatellite. A minisatellite is a tract of repetitive DNA in which certain DNA motifs (10â€“60 base pairs) are typically repeated 5-50 times. They occur at ~ 1,000 locations in the human genome,  are subject to high mutation  and have a high diversity in the population. They are prominent in the centromeres and telomeres. The name "satellite" comes from the fact that they cause a separate layer during centrifuging. They are also called VNTR (variable number of tandem repeats). Microsatellites are referred to as short tandem repeats (STRs) or simple sequence repeats (SSRs). 
* [Human Genome](https://en.wikipedia.org/wiki/Human_genome): As an example of the details, Chr 1 is 8.5 cm long with 248956422 bps (~249 Mpb). The length of a bp is 340 pm (~250 M x 340 p = 0.25 x 0.34 m = 2.5 x 3.4 cm = 8.5 cm). It has 12 M variations (genetic markers of 1+ length), 2058 protein coding genes, 1220 psedogenes, 1200 long ncRNA, and 496 small ncRNA. Long non-coding RNAs (lncRNA) are  > 200b with no protein-coding potential. These incluce rRNA (ribosomal) and other long RNAs involved in gene expression regulation and epigenetic modifications of DNA nucleotides and histone proteins. Small ncRNA are <200b and include miRNA (micro) and snoRNA (Small nucleolar).
* [Protein-coding genes](https://en.wikipedia.org/wiki/Human_genome): The median size of a protein-coding gene is 26,288 bp (mean = 66,577 bp). The median size of an exon is 122 bp (mean = 145 bp), the median number of exons is 7 (mean = 8.8), and the median coding sequence encodes 367 amino acids (mean = 447 amino acids).
* Linkage disequilibrium (LD) is the non-random association of alleles at different loci in a given population.
* A haplotype is a set of DNA variations, or polymorphisms, that tend to be inherited together. A haplotype can refer to a combination of alleles or to a set of single nucleotide polymorphisms (SNPs) found on the same chromosome. As an example, if there are 3 SNPs in a DNA stretch, there are 8 possible SNP combinations, and hence 8 haplotypes. Explained [here.](http://learn.genetics.utah.edu/content/precision/snips/) Only some of these may actually exist. A child will get a haplotype pair from the parents. The ref describes with the example of Albetrol used to relieve Asthma symtoms. In a 3000 bp stretch around the ADRB2 gene, scientists found 13 SNPs and 12 haplotypes. Of these, 5 haplotype pairs are more common, and the response is different for each of these pairs, from none to verygood. So, the doctor can one day use this haplotype pair info to prescribe or not Albeterol. *this info is useful for the more common haplotypes. For rarer haplotypes, there is no information to either use or not use Albeterol*. 

Refs:
* Hirschhorn, J.N., and Daly, M.J.,  Genome-Wide Association Studies for common diseases and complex traits, *Nature Reviews Genetics*, vol. 6, Feb 2005, pp. 95-108
* Cookson, W., et al., Mapping complex disease traits with global gene expression, *Nature Reivews Genetics*, Vol. 10, March 2009, pp. 184-194.
* Frazer, K.A., et al., Human Genetic variation and its contribution to complex traits, *Nature Reviews Genetics*, vol. 10, April 2009, pp. 241-251
* Stephens, M., and Balding, D.J., Bayesian statistical methods for genetic association studies, *Nature Reviews Genetics*, vol. 10, October 2009, pp. 681-690.
* Civelek, M., and Lusis, A.J., Systems genetics approaches to understand complex traits, *Nature Reviews Genetics*, vol. 15, Janauary 2014, pp. 34-48.
* Ott, J., et al., Genetic linkage analysis in the age of Whole-genome sequencing, *Nature Reviews Genetics*, vol. 16, May 2015, pp. 275-284.

