**Alu elements** are short repititive DNA sequences (~300 bp long) and represent the most abundant transposable elements (~1 million copies, or 10% of the DNA) that are dispersed throughout the human genome. They are also called selfish or parasitic genes because their only function is to self-reproduce. They are classified as SINEs (Short interspersed nuclear elements), a subclass of repititive DNA elements. Alu insertions play a role in a few inherited human diseases (relevant ones: familial hypercholosteroiemia, hemophilia, and diabetes mellitus type II) and in various forms of cancer. See [wiki](https://en.wikipedia.org/wiki/Alu_element). Alu elements are highly conserved within primate genomes. 

Considered earlier as 'junk DNA', they are now recognized to play diverse roles in gene regulation and genome evolution. The Alu element has been a major source of new exons during primate evolution, created 60 mya (1). The consensus sequence of Alu comprise sequences that are similar to the 5' and the 3' splice site signals. After insertion of an Alu element into the intronic region of an existing gene, subsequent mutations could utilize these splice sites or introduce new splicing regulatory signals, leading to the creation of a new exon. Most Alu exons are rarely incorporated into transcripts. Only a small number of Alu exons have high splicing activity, along with tissue specificity. Shen et al. (1) carried out a genome-wide analysis of Alu exon splicing using transcriptome profiles generated by deep RNA sequencing (RNA-Seq). RNA-seq, unlike the previous EST and exon array technologies, permits the inclusion of all exons dispersed throughout the genome in an unbiased manner. 

Thousands of human genes contain spliced exons derived from Alu elements. Some of these exons may have acquired genuine biological functions. Shen et al. analyzed the transcript inclusion level in the cerebellum for inclusion of such exons. For zinc finger protein 445 (*ZNF445*) this estimated level was 79%. The authors also point out that this protein is found with signficant inclusion level in other tissues as well (heart, kidney, liver, muscle, etc.). They also conclude that Alu exon enrichment is a more recent phenomenon, in terms of human evolution, being mapped to the primate age, the youngest phylogenetic age. These 5'-UTR Alu exons significantly regulate translational efficiency (up or down) through a variety of molecular mechanisms. This suggests an interesting evolutionary scenario: that the creation and establishment of Alu exons could impact the primate and human transcriptomes globally by fine-tuning the protein production of master transcriptional regulators (1). 

Kim et al. (2) indicates that there is a 'master gene' model that is generally accepted. The master gene mutated to create different subfamilies of *Alu* elements and they are amplified from the hyperactive master genes. *Alu* elements are associated with about 0.1% of human genetic disorders (inherited diseases, and blood and neurological disorders). The authors mention another study that reported that intronic insertion of *AluYb9* causes exon  skipping and leads to hemophilia A disease, which is an X-linked disorder caused by exon skipping of exon 19  (by this *Alu* in intron 18) in Factor VIII gene (Note: This is a LINE-1, not a SINE type). 515 *Alu*-mediated deletion events occurred in the human genome after the divergence of the human and chimpanzee lineages. Of these, 24 were AIMDs (*Alu* insertion-mediation deletions) and 492 were ARMDs (*Alu* recombination-mediated deletions). Recombination rate is high because of a high level of sequence identity and *Alu* abundance. Although *AluJ* subfamily is more abundant than *AluY* subfamily, the latter has more ARMDs. Example from Table 1: *ACE* Gene in Chr 17 with *AluYa5* insertion leads to Alzheimer's disease. *BMPR2* Gene in Chr 2 with *AluY* ARMD_NAHR (nonallelic homologous recombination) causes pulmonary arterial hypertension. 

Payer et al. (3) explore polymorphic *Alu* insertion variants that function in a common variant, common disease paradigm. They cataloged 809 polymorphic *Alu* elements mapping to 1,159 loci implicated in disease by GWAS. They found that *Alu* insertion variants occur disproportionately at GWAS loci. They found 44 of these *Alu* elements in linkage disequilibrium (LD) with the trait-associated SNP. This represents a 20 fold increase in the number of polymorphic *Alu* elements associated with human phenotypes. To identify polymorphic retrotransposon copies that could affect disease risk, they focused on those copies mapping to locations already associated with disease phenotypes by GWAS. Earlier, they catalogued polymorphic *Alu* elements near GWAS signals. By using conditions specific to detecting  *Alu*Ya5/8 and *Alu*Yb8/9 families, they avoided the evolutionarily older *Alu* subfamilies that are largely invariant for all of the human population (i.e., homozygously present in all people). Their work is based on 9 Utal residents. 

Mapping SNPs to molecular process is crucial for understanding disease. Brodie et al. (4) discuss a new way to associate GWAS-identified SNPs to causative genes.  Most such SNPs are in non-coding regions and far away from the genes they implicate. They used a biological pathway-based approach to find this linkage. The affected genes were often found to be up to 2 Mbps away from the associated SNP,  not necessarily the closest genes to the SNP. Existing approaches for mapping SNPs to genes  reveal only 86 significant phenotype-pathway associations for all known GWAS hits combined. There are 3 such approaches: LD (Linkage Disequilibrium);  an upstream cutoff of 2 Kbps and a downstream cutoff of 0.5 Kbps to map a SNP to a gene (as with SNP database dbSNP); and  A larger cutoffs of up to 100 Kbps or more at 500 Kbps. But even these cutoffs leave many GWAS hits with no associations to any gene.

Using the biological pathway-based approach they mapped virtually all SNPs to genes and revealed 435 statistically significant phenotype-pathway associations. They also found that SNPs that are mapped to distant genes have significantly more **large insertions/deletions** around them than other SNPs, suggesting that these SNPs may sometimes be markers for large insertions/deletions that may affect large genomic regions and distant genes. 

Brodie's method: They extracted Phenotype-SNP associations  from GWAS data in the NHGRI GWAS catalog. It contains manually curated entries of published GWAS, in which SNPs were associated with diseases, phenotypes, and traits.  Gene symbols were taken from Genenames, while the genomic locations of SNPs and genes were taken from the UCSC genome browser. Biological pathways and their associated genes were taken from the KEGG pathway database (and cross-validated with  ConsensusPathDB, or CPDB).  Genomic indels were taken from DGV, a database of genomic structure variants (SV). These were used for the analysis of whether more indels fall in phenotype-associated SNP-gene regions. For the analysis of indels around SG regions (the region between a SNP and a gene) they used GWAS Catalog version downloaded from www.genome.gov/gwastudies.

References:
1. Shen et al., (2001). Widespread establishment and regulatory impact of Alu exons in human genes, *PNAS*, Vol. 108, pp. 2837-2842.
2. Kim et al., (2016).  Structural Variation of *Alu* element and human disease, *Genomics and Informatics*, Vol. 14, pp. 70-77.
3. Payer, et al., (2017). Structural variants caused by *Alu* insertions are associated with risks for many human diseases, *PNAS*, published online, pp. E3984-E3992 
4. Brodie et al. (2016). How far from the SNP may the causative genes be? *Nucleic Acids Research*, Vol. 44, Pp. 6046â€“6054, accessed [here](https://academic.oup.com/nar/article/44/13/6046/2457612). 

Glossary: 
* EST - expressed sequence tag
* GWAS - Genomic Wide Association Studies
* LINE-1 - long interspersed nuclear element-1
* SINE - Short interspersed nuclear element
* TAS - trait-associated SNP
* LD - Linkage Disequilibrium

Databases: 
* KEGG - http://www.genome.jp/kegg/ - biological pathways: molecular networks involving diseases and drugs. 
