**Alu elements** are short repititive DNA sequences (~300 bp long) and represent the most abundant transposable elements (~1 million copies, or 10% of the DNA) that are dispersed throughout the human genome. They are also called selfish or parasitic genes because their only function is to self-reproduce. They are classified as SINEs (Short interspersed nuclear elements), a subclass of repititive DNA elements. Alu insertions play a role in a few inherited human diseases (relevant ones: familial hypercholosteroiemia, hemophilia, and diabetes mellitus type II) and in various forms of cancer. See [wiki](https://en.wikipedia.org/wiki/Alu_element). Alu elements are highly conserved within primate genomes. 

Considered earlier as 'junk DNA', they are now recognized to play diverse roles in gene regulation and genome evolution. The Alu element has been a major source of new exons during primate evolution, created 60 mya (1). The consensus sequence of Alu comprise sequences that are similar to the 5' and the 3' splice site signals. After insertion of an Alu element into the intronic region of an existing gene, subsequent mutations could utilize these splice sites or introduce new splicing regulatory signals, leading to the creation of a new exon. Most Alu exons are rarely incorporated into transcripts. Only a small number of Alu exons have high splicing activity, along with tissue specificity. Shen et al. (1) carried out a genome-wide analysis of Alu exon splicing using transcriptome profiles generated by deep RNA sequencing (RNA-Seq). RNA-seq, unlike the previous EST and exon array technologies, permits the inclusion of all exons dispersed throughout the genome in an unbiased manner. 

Thousands of human genes contain spliced exons derived from Alu elements. Some of these exons may have acquired genuine biological functions. Shen et al. analyzed the transcript inclusion level in the cerebellum for inclusion of such exons. For zinc finger protein 445 (*ZNF445*) this estimated level was 79%. The authors also point out that this protein is found with signficant inclusion level in other tissues as well (heart, kidney, liver, muscle, etc.). They also conclude that Alu exon enrichment is a more recent phenomenon, in terms of human evolution, being mapped to the primate age, the youngest phylogenetic age. These 5'-UTR Alu exons significantly regulate translational efficiency (up or down) through a variety of molecular mechanisms. This suggests an interesting evolutionary scenario: that the creation and establishment of Alu exons could impact the primate and human transcriptomes globally by fine-tuning the protein production of master transcriptional regulators (1). 




References:
1. Shen et al., (2001). Widespread establishment and regulatory impact of Alu exons in human genes, *PNAS*, Vol. 108, pp. 2837-2842.
2. Kim et al., (2016).  Structural Variation of *Alu* element and human disease, *Genomics and Informatics*, Vol. 14, pp. 70-77.
3. Payer, et al., (2017). Structural variants caused by *Alu* insertions are associated with risks for many human diseases, *PNAS*, published online, pp. E3984-E3992 



Glossary: 
* expressed sequence tag (EST)
